packages/contracts-core/contracts/create3/CREATE3Factory.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0
```

Comment:
I think this contract belongs to a submodule, as it's not relevant to the whole system.

Could use this repo instead of solmate as submodule: https://github.com/ZeframLou/create3-factory

Reply:
Okay, will do

Reply:
#1387 will address